app/view/registration/RegistrationController.js

```diff
@@ -9,16 +9,16 @@ Ext.define('MoMo.login.view.registration.RegistrationController', {
         'BasiGX.util.Url'
     ],
 
-    onTermCheckChange: function(box, newVal){
-        var form = box.up('form').getForm(),
-            button = box.up('form').down('button[name=createAccountButton]');
-
-        if (newVal && form.isValid()) {
-            button.enable();
-        } else {
-            button.disable();
-        }
-    },
+//    onTermCheckChange: function(box, newVal){
+//        var form = box.up('form').getForm(),
+//            button = box.up('form').down('button[name=createAccountButton]');
+//
+//        if (newVal && form.isValid()) {
+//            button.enable();
+//        } else {
+//            button.disable();
+//        }
+//    },
```

Comment:
If Checkbox will be removed in view `Registration.js`, can this be removed as well?